### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by neogopher**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind documentation

Closes ENG-1989
Closes ENG-1990

### CODE REVIEW

File: docs/pages/what-are-virtual-clusters.mdx [hunk 1]
```diff
@@ -10,9 +10,9 @@ Virtual clusters are fully working Kubernetes clusters that run on top of other
   <figcaption>vcluster - Architecture</figcaption>
```
Comment (alexandradragodan) [inline docs/pages/what-are-virtual-clusters.mdx:10]: Change to 'vCluster' in figcaption here

File: docs/pages/networking/integrated_coredns.mdx [hunk 2]
```diff
@@ -3,39 +3,39 @@ title: Integrated CoreDNS
...
 
 ### Advanced Configuration
 
 For more advanced configuration of the integrated CoreDNS itself follow the below sections
 
 #### Custom CoreDNS Configuration
 
-vcluster.pro provides and option to pass in your custom CoreDNS configuration which can be passed to extend the default one used by vcluster.
-To utilize this, one needs to pass the `Corefile` configuration as part of a configmap named `coredns-custom` in the same namespace as the vcluster
+vCluster.Pro provides and option to pass in your custom CoreDNS configuration which can be passed to extend the default one used by vCluster.
```
[the remark this answers is not in the document]
Comment (neogopher) [reply docs/pages/networking/integrated_coredns.mdx]: Updated.

File: docs/pages/using-vclusters/access.mdx [hunk 3]
```diff
@@ -230,7 +230,7 @@ Create the resource in the namespace via:
 kubectl apply -f nodeport.yaml
 ```
 
-Find out the external port via `kubectl get svc vcluster-nodeport -n my-vcluster`:
+Find out the external port via `kubectl get svc vCluster-nodeport -n my-vcluster`:
```
[the remark this answers is not in the document]
Comment (neogopher) [reply docs/pages/using-vclusters/access.mdx]: Yeah. Fixed.

File: docs/pages/what-are-virtual-clusters.mdx (hunk 1 above)
Comment (neogopher) [reply docs/pages/what-are-virtual-clusters.mdx]: Done.

**Review by alexandradragodan — APPROVED** [review]
looks good. lgtm

**Pull request merged**